recipes/glidertest/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "glidertest" %}
+{% set version = "0.0.1a6" %}
```

Comment:
Are you planning on a stable release anytime soon? If so, we can hold this off and publish that. If not you will have to follow cfep-05: https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Yes! I did that in the wrong order. Stable v0.0.1 now on PyPI and used in this recipe